Template/protocol_template.py

```diff
@@ -0,0 +1,207 @@
+from opentrons import protocol_api
+
+metadata = {
+    'protocolName': 'Protocol Title',
+    'author': 'AUTHOR NAME <authoremail@company.com>',
+    'source': 'Custom Protocol Request',
+    'apiLevel': '2.11'   # CHECK IF YOUR API LEVEL HERE IS UP TO DATE
+                         # IN SECTION 5.2 OF THE APIV2 "VERSIONING"
+}
+
+
+def run(ctx: protocol_api.ProtocolContext):
+
+    [
+     custom_variable1,
+     custom_variable2
+    ] = get_values(  # noqa: F821 (<--- DO NOT REMOVE!)
+        "custom_variable1",
+        "custom_variable2")
+
+    if not 1 <= custom_variable1 <= 12:
+        raise Exception("Enter a value between 1-12")
+
+    # define all custom variables above here with descriptions:
+
+    # number of samples
+    custom_variable1 = custom_variable1
```

Comment:
I might doing something like:
`custom_variable1 = _custom_variable1`

But open to discussion

Reply:
That's a good idea, that makes it more explicit that we're assigning some input variable to a variable that we will use in our script.

Also the comment could say "e.g. number of samples" instead